monitor/dash_demo.py

```diff
@@ -1,7 +1,7 @@
 import dash_bootstrap_components as dbc
 import dash_core_components as dcc
 import dash_html_components as html
-import requests
+import requests  # type: ignore
```

Comment:
I suggest going with the following option in config file instead

https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-library-stubs-or-py-typed-marker

<img width="771" alt="image" src="https://github.com/ScilifelabDataCentre/stackn/assets/8545082/cba6ab39-05f3-44c3-8296-8066431a963c">

Reply:
Missing imports are already ignored globally in the pyproject.toml config file using

[tool.mypy]
ignore_missing_imports = true

The reason for these #type ignore lines are to silence the warning Library stubs not installed for "requests"  [import]

Reply:
Aha

But I still think that it would be better to explicitly silence libraries instead of global ignore.

It would make developer think to look for stubs after addition of a new library.

Reply:
I agree. But I cannot seem to get ignores per library to work in the .toml config file. You?

Reply:
This worked for me

```toml
[tool.mypy]
strict = false
python_version = "3.8"
warn_return_any = true
exclude = ["venv", ".venv", "migrations", "examples"]

[[tool.mypy.overrides]]
module = [
	"django.*",
	"rest_framework.*",
	"flatten_json.*",
	"guardian.*",
	"django_filters.*",
	"rest_framework_nested.*",
	"tagulous.*",
	"minio.*",
	"celery.*",
	"dash.*",
	"dash_core_components.*",
	"dash_html_components.*",
	"django_plotly_dash.*",
	"dash_bootstrap_components.*",
	"s3fs.*",
	"setuptools.*",
	]
ignore_missing_imports = true

Reply:
Thank you. I will submit a follow-up PR. Great find :-)